### ISSUE TITLE
[Android][Inconsistency with desktop] [Date input component does not honour the default value]

### ISSUE THREAD

**Issue by shbhadra** [microsoft/AdaptiveCards#3671]
# Platform
* Android

# Author or host
Host

# Version of SDK
1.2.5

# Details
The date input component does not honour the default value and always shows the current date by default. For the card as in the attached json, following are the views on desktop and android.
(https://github.com/microsoft/AdaptiveCards/files/3944744/default_date.json.zip)

 View on desktop
![image](https://user-images.githubusercontent.com/51158485/70527594-4a993e00-1b72-11ea-8077-2fb71230326c.png)
View on android
<img width="320" alt="Screenshot 2019-12-10 at 5 33 21 PM" src="https://user-images.githubusercontent.com/51158485/70528004-3570df00-1b73-11ea-919f-ccaf0970a29b.png">
[default_date.json.zip](https://github.com/microsoft/AdaptiveCards/files/3944750/default_date.json.zip)

```json
{
   "type":"AdaptiveCard",
   "body":[
      {
         "type":"TextBlock",
         "size":"medium",
         "weight":"bolder",
         "text":"Test title"
      },
      {
         "type":"TextBlock",
         "text":"Testing Date input.",
         "wrap":true
      },
      {
         "type":"Input.Date",
         "value":"03/12/2019",
         "id":"date"
      }
   ],
   "$schema":"http://adaptivecards.io/schemas/adaptive-card.json",
   "version":"1.0"
}
```

**Comment by nesalang** [comment]
Time and Date value have to be expressed in specific format.
Date: YYYY-MM-DD
Time:HH:MM
Resolution: Created [PR](https://github.com/microsoft/AdaptiveCards/pull/3706) for updated doc

**Comment by shalinijoshi19** [comment]
@nesalang,  thanks! Are we now consistent across all renderers/platforms wrt time and date expressions?

**Comment by shbhadra** [comment]
@nesalang @shalinijoshi19 I believe this has been erroneously closed, the issue still persists in the 1.2.6 drop. Note that the bug is not regarding the format of the date, but that the date input element picks up todays date by default, instead of the specified date value.

**Comment by nesalang** [comment]
@shbhadra ,

Thanks for reaching out to us.

'''json
{
         "type":"Input.Date",
         "value":"**03/12/2019**",
         "id":"date"
      }
'''

The date string, "03/12/2019",  is in wrong format. 
The correct format is 2019-03-12 . i.e YYYY-MM-DD.
